spec/GraphQLOverHTTP.md

```diff
@@ -112,72 +169,133 @@ http://example.com/product/graphql
+somewhere in the request chain). For this reason, this specification introduces
+the `application/graphql+json` media type; however, to give existing servers
+time to move over, it is not required to be supported until 1st January 2025.
 
 # Request
 
-A server MUST accept POST requests, and MAY accept other HTTP methods, such as GET.
+A server MUST accept POST requests, and MAY accept other HTTP methods, such as
+GET.
 
 ## Request Parameters
 
-A request for execution should contain the following request parameters:
+A {GraphQL-over-HTTP request} is formed of the following parameters:
+
+- {query} - A Document containing GraphQL Operations and Fragments to execute.
+- {operationName} - (_Optional_): The name of the Operation in the Document to
+  execute.
+- {variables} - (_Optional_): Values for any Variables defined by the Operation.
+- {extensions} - (_Optional_): This entry is reserved for implementors to extend
+  the protocol however they see fit.
 
-* {query} - A Document containing GraphQL Operations and Fragments to execute.
-* {operationName} - (*Optional*): The name of the Operation in the Document to execute.
-* {variables} - (*Optional*): Values for any Variables defined by the Operation.
-* {extensions} - (*Optional*): This entry is reserved for implementors to extend the protocol however they see fit.
+Note: When comparing against
+[request in the GraphQL specification](https://spec.graphql.org/draft/#request)
+you should note the schema and "initial value" are not included in the
+GraphQL-over-HTTP request, they are handled by the server based on the URL used.
 
-Note: Be aware that `query` is a misleading name as it can contain a string describing multiple operations, each of which may be a query, mutation or subscription. A better name would have been `document`, but the term `query` is well established.
+Note: Be aware that `query` is a misleading parameter name as its value is a
+string describing one or more operations, each of which may be a query, mutation
+or subscription. A better name would have been `document`, but the term `query`
+is well established.
 
-Note: depending on the serialization format used, values of the aforementioned parameters can be
-encoded differently but their names and semantics must stay the same.
+Note: Depending on the serialization format used, values of the aforementioned
+parameters can be encoded differently but their names and semantics must stay
+the same.
 
-Note: specifying `null` in JSON (or equivalent values in other formats) as values for optional request parameters is equivalent to not specifying them at all.
+Note: Specifying `null` in JSON (or equivalent values in other formats) as
+values for optional request parameters is equivalent to not specifying them at
+all.
 
-Note: {variables} and {extensions}, if set, must have a map as its value.
+Note: {variables} and {extensions}, if set, must have a map as their value.
+
+### JSON Encoding
```

Comment:
Maybe, but I moved it up here because it's one of a number of possible encodings of the request parameters; for `GET` there's only really one choice. But yeah, maybe we should move it back again :man_shrugging:

Reply:
Moved.